scripts/setup_gemdaq.sh

```diff
@@ -207,41 +288,58 @@ fi
 
 # Setup locations
 ####################
-if [[ $SYSTEM_INFO == *"lxplus"* ]];
-then
-    export DATA_PATH=/afs/cern.ch/work/$USER[0,1]/$USER/CMS_GEM/Data/gemdata
-elif [[ $SYSTEM_INFO == *"gem904"* ]];
+if [[ -z $DATA_PATH ]]; # Don't override existing value
```

Comment:
Revert this change.  This requirement *is* problematic since if `DATA_PATH` is set by a user on a `904` machine or a `p5` machine additional necessary `env` variables will *not* be set.

If you want to implement this check to not over-write the `DATA_PATH` variable then do so *only* on `lxplus`.  For the P5 and 904 machines a user should not be using a different `DATA_PATH` location since data taken/analysis performed on those test stands/p5 should be placed in the centrally defined location given below.

Reply:
> Additionally in the case of lxplus add an if statement that if the path does not exist it is made with a mkdir -p $DATA_PATH command.

I disagree. Rationale:

If `$DATA_PATH` doesn't exist when the user sets up its environment, I can see two cases:

1. Either the data is not where the user expects it to be (not downloaded yet, filesystem not mounted, erased...)
2. Or  `$DATA_PATH` is wrong (or not set yet).

In either case user intervention is required, so it's good to display a warning. Creating an empty directory will *not* solve any of the problems; worse, it will silence the warning.

Reply:
> Please implement the creation of an empty directory. This gives a USER a prime place to navigate too and place the data.

Done. Will maintain a local patch not to have it in my own version.